pytest.ini

```
@@ -19,6 +19,9 @@ addopts =
     --cov-report term-missing:skip-covered
     --cov-report xml:.test-results/pytest/cov.xml
     --no-cov-on-fail
+
+    # mention where to look for tests
+    --pyargs test ansiblelint.rules
```

Putting these on the same line triggers a misleading suggestion that `test ansiblelint.rules` are the args to `--pyargs` option that is not true:
```console
  --pyargs              try to interpret all arguments as python packages.
```
So we must separate the test lookup targets:
```suggestion
    # interpret all the target args as importables:
    --pyargs

    # importable packages for test lookup:
    test ansiblelint.rules
```